README.adoc

```diff
@@ -1533,6 +1533,47 @@ Time.zone.now # => Fri, 12 Mar 2014 22:04:47 EET +02:00
 Time.current # Same thing but shorter.
 ----
 
+== Duration
+
+Don't use `.since`, `.after` without a parameter, prefer `.from_now`.
+
+Dont't use `.until`, `.before`, without a parameter, prefer `ago`.
+
+Dont't use `.ago' or `.from_now` with a parameter, prefer their respective aliases `.until` or `.before` for `ago` and `.since` or `.after` for `from_now`
+
+Don't use hard coded negative numbers for the duration subject
+
+As a general rule, make sure that reading the code as it was plain english makes total sense.
+
+[source,ruby]
+----
+# bad
+5.hours.since # => It's not clear at first sight that `since` refers to "since now" 
```

Comment:
`=>` does not make sense in this context. In usually tells what the output of the statement is.

Reply:
yeah, copy pasted from other examples, I'm going to rewrite everything